awx/ui_next/src/components/CodeMirrorInput/VariablesDetail.jsx

```diff
@@ -62,7 +58,7 @@ function VariablesDetail({ value, label, rows }) {
       >
         <CodeMirrorInput
           mode={mode}
-          value={currentValue}
+          value={currentValue || '---'}
```

Comment:
Should we be calling one of the json/yaml utils here? https://github.com/ansible/awx/blob/devel/awx/ui_next/src/util/yaml.js#L16

Reply:
update: so I ended up taking a few seconds to check and see if these utils could be used to set up this initial value. I see a json `parsing` error so it doesn't appear that we can. 

With that out of the way, this PR LGTM :+1: